recipe/meta.yaml

```diff
@@ -10,11 +10,11 @@ build:
 
 requirements:
   run:
-    - pangeo-dask =2020.11.05
+    - pangeo-dask =2020.12.11
     - dask-labextension =3.0.0
     - ipywidgets =7.5.1
     - jupyter-server-proxy =1.5.0
-    - jupyterhub-singleuser =1.2.1
+    - jupyterhub-singleuser =1.3.0
```

Comment:
Question for @consideratio or @yuvipanda. Something i've never quite understood. Is jupyterhub-singleuser necessary in our pangeo notebook image and should the version match the jupyterhub version we're running? Following the version path for pangeo hubs we're running daskhub 4.5.6 --> jupyterhub helm chart 0.10.6 --> jupyterhub 1.2.2
(https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/0.10.6/chartpress.yaml)

Reply:
No worries in this case im fairly confident. If can be an issue if there are some breaking change in how the hub/user server interact.

Overall, im not sure, but i would say we shouldnt worry here about it.